src/ml/neural_net/mps_layer_instance_norm_data_loader.mm

```diff
@@ -95,15 +95,15 @@ - (instancetype) initWithParams:(NSString *)name
                                                    dataType:(MPSDataTypeFloat32)];
 
     _style_props = [[NSMutableArray alloc] init];
-
+    
     for (NSUInteger index = 0; index < styles; index ++) {
       TCMPSInstanceNormDataLoaderProps *style_property = [[TCMPSInstanceNormDataLoaderProps alloc] init];
 
-      id<MTLBuffer> gammaBuffer = [dev newBufferWithBytes:_gamma_weights.mutableBytes
+      id<MTLBuffer> gammaBuffer = [dev newBufferWithBytes:_gamma_weights.bytes + (index * _numberOfFeatureChannels * sizeof(float))
```

Comment:
For readability, could we just factor out the computation of the offset into something like
```
size_t offset = ...;
```